components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2783,4 +2820,103 @@ void RewardsServiceImpl::OnDeleteActivityInfo(
   }
 }
 
+void RewardsServiceImpl::GetAutoContributeCount(
+    const rewards_counter::OnRewardsCountedCallback callback) {
+  GetReconcileStamp(
+      base::Bind(&RewardsServiceImpl::GetCountByReconcileStamp,
+        AsWeakPtr(), callback));
+}
+
+void RewardsServiceImpl::GetLastReconcileStamp(
+    const rewards_counter::OnRewardsCountedCallback callback,
+    int64_t count) {
+  base::PostTaskAndReplyWithResult(
+      file_task_runner_.get(),
+      FROM_HERE,
+      base::Bind(&AutoContributionLastReconcileDateOnFileTaskRunner,
+          publisher_info_backend_.get()),
+      base::Bind(&RewardsServiceImpl::OnRewardsCounted,
+          AsWeakPtr(),
+          callback,
+          count));
+}
+
+void RewardsServiceImpl::GetCountByReconcileStamp(
+    const rewards_counter::OnRewardsCountedCallback callback,
+    uint64_t reconcile_stamp) {
+  base::PostTaskAndReplyWithResult(
+      file_task_runner_.get(),
+      FROM_HERE,
+      base::Bind(&AutoContributionCountOnFileTaskRunner,
```

Comment:
Naming should be consistent so this should be `GetAutoContributeCountOnFileTaskRunner`

Reply:
Fixed